### PR TITLE
[Doc][Docker] fix typo on start with docker

### LINKED ISSUE #15533: [Doc][Docker] fix typo on start with docker
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

change database in `SPRING_DATASOURCE_URL` into dolphinscheduler

### Documentation Links

https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/zh/guide/start/docker.md?plain=1

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by pegasas**
## Purpose of the pull request

fix https://github.com/apache/dolphinscheduler/issues/15533

## Brief change log

fix type on docker start guide.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: docs/docs/en/guide/start/docker.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ $ DOLPHINSCHEDULER_VERSION=<version>
 # Initialize the database, make sure database <DATABASE> already exists
 $ docker run -d --name dolphinscheduler-tools \
     -e DATABASE="postgresql" \
-    -e SPRING_DATASOURCE_URL="jdbc:postgresql://localhost:5432/<DATABASE>" \
+    -e SPRING_DATASOURCE_URL="jdbc:postgresql://localhost:5432/dolphinscheduler" \
```
Comment (SbloodyS) [inline docs/docs/en/guide/start/docker.md]: I don't think this is a typo since we already have comment above.

Comment (pegasas) [reply docs/docs/en/guide/start/docker.md]: So I think maybe we should correct below 4 into DATABASE?
I think we should all changed to dolphinscheduler or <DATABASE>.
or else it is confusing.
It is also a little bit confuse with SPRING_DATABASE configuration.

Comment (zhongjiajie) [reply docs/docs/en/guide/start/docker.md]: change others components env `SPRING_DATASOURCE_URL` LGTM

Comment (pegasas) [reply docs/docs/en/guide/start/docker.md]: > change others components env `SPRING_DATASOURCE_URL` LGTM

Thanks for discussion.
Updated.

**Review by SbloodyS — COMMENTED** [review]
LGTM.

**Review by SbloodyS — APPROVED** [review]

**Pull request merged**